lens-cube/src/test/java/org/apache/lens/cube/parse/TestCubeRewriter.java

```diff
@@ -411,7 +412,7 @@ public void testPartColAsQueryColumn() throws Exception {
       "select cubestate.name, cubestate.countryid, msr2 from" + " testCube" + " where cubestate.countryid = 5 and "
         + TWO_DAYS_RANGE, conf);
     expected =
-      getExpectedQuery(TEST_CUBE_NAME, "select cubestate.name, cubestate.countryid, sum(testcube.msr2)" + " FROM ",
+      getExpectedQuery(TEST_CUBE_NAME, "select cubestate.name, cubestate.countryid , sum(testcube.msr2)" + " FROM ",
```

Comment:
Why this formatting change ?  Original one looks fine to me . 

Please check all other formatting changes too